Here is patch without --- and +++, but with "diff --git" line.
diff --git filename filename
@@ -5,7 +5,7 @@ foo
 aaa
 bbb
 ccc
-ddd
+eee
 fff
 ggg
 hhh
